ci/azure-pipelines.yml

```diff
@@ -114,6 +114,26 @@ stages:
             displayName: Build on Ubuntu
             condition: contains(variables['build_container'], 'ubuntu')
 
+          - bash: |
+              set -eEx
+              test -f Makefile && make -i distclean || true
+              ./autogen.sh
+              export CFLAGS='-Werror -Wall'
+              ./configure --disable-kernel-module
+              make
+            displayName: Build on CentOS without kernel module
+            condition: contains(variables['build_container'], 'centos')
+
+          - bash: |
+              set -eEx
+              test -f Makefile && make -i distclean || true
+              ./autogen.sh
+              export CFLAGS='-Werror -Wall'
+              ./configure --enable-gtest --disable-kernel-module
```

Comment:
why do we build with gtests on ubuntu but without them on centos?

Reply:
Intent is to build with and without gtest, with disabled kernel module. As distribution should not have impact we did not aim to test all combinations on all distributions.